src/components/App/App.jsx

```diff
@@ -28,6 +29,7 @@ export default function App(){
 				<About />
 				<Characters />
 				<Episodes />
+				<EpisodePreview />
```

Comment:
rather than having the `<EpisodePreview>` as another 'page', I think we discussed it as being a child of the `<Episode>` component, with the thumbnail triggering the full-screen mode. Something like:

```
<Episode>
    <h1>
        Episode 1: the whatever part 1
    </h1>
    <p>
        Synopsis
    </p>
    <EpisodePreview /> // the thumbnail
</Episode>
```

Reply:
Done